src/parser/mod.rs

```diff
@@ -649,6 +649,7 @@ impl<'a> Parser<'a> {
                     self.prev_token();
                     self.parse_export_data()
                 }
+                Keyword::VACUUM => self.parse_vacuum(),
```

Comment:
```suggestion
                Keyword::VACUUM => {
                    self.prev_token();
                    self.parse_vacuum()
                }
```
Thinking so that we can make `parse_vacuum` standalone to be able to parse a full `VACUUM` statement